docs/dev/index.md

```diff
@@ -1 +1,45 @@
-[IntelliJ Setup](IntelliJ/) 
\ No newline at end of file
+# PatternPedia Developer Guide
+This document provides an index to all development guidelines and background information of the PatternPedia.
+- [ADR](/adr) - Information on Architectural decisions can be found here
+
+## Quick Develop
+
+1. Clone the repository `git clone https://github.com/PatternPedia/pattern-pedia-views-backend.git`.
+2. Build the repository `mvn package -DskipTests` (skiping the tests for a faster build), Java 8 required.
+3. Clone the repository `git clone https://github.com/PatternPedia/pattern-pedia-views-ui.git`.
+4. Build the repository `mvn package -DskipTests` (skiping the tests for a faster build), Java 8 required.
+5. Continue your IDE setup:
+    - [IntelliJ Ultimate](/IntelliJ/)
+6. Set up database:
+    - Open Terminal in IntelliJ
+    ![checkstyle](IntelliJ/graphics/terminal.png)
+    - Navigate to directory  ``..\.docker\``
+    - Insert the following commands ``docker-compose up -d``
+    - Open "Database"
+    ![checkstyle](IntelliJ/graphics/open-database.png)
+    - Click the "+"
+    - Go to "Data Source > PostgresSQL"
+    - Enter username and password: "postgres" (both)
+    ![checkstyle](IntelliJ/graphics/postgres-setup.png)
+    - Click "Apply"
+    
+
+7. Start the application (via the runconfig that you configured in step 5)
+
+## Main API Endpoints
+API-Root: /pattern-pedia-views-backend (?)
+
+Swagger-Documentation: /swagger-ui (?)
+
+HAL - Browser: /explorer (?)
```

Comment:
Swagger-UI: (Swagger is more than just a documentation as you can insert data here) http://localhost:8080/swagger-ui/index.html?url=/v3/api-docs#/

Hal-Browser: On "/" -> redirects to http://localhost:8080/explorer/index.html#uri=/